packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1390,6 +1390,24 @@ export function wrapTextInsideDonut(selectorClass: string, maxWidth: number) {
   });
 }
 
+export function shouldWrapContent(content: string, id: string, maxWidth: number) {
```

Comment:
This function has already been added in https://github.com/microsoft/fluentui/commit/ddffebd7abd2e0c80b31300d61f738c6cf0d1e02#diff-bb4700f7d37665df6f440f23cd3b39fbf12ab0f26184bb52a51111535a1f3562 commit.
You can use that instead.

Reply:
used the existing functionality